lib/reindex/s3_bucketd.py

```diff
@@ -0,0 +1,203 @@
+import requests
+import redis
+import json
+import ast
+import sys
+import time
+import urllib
+import re
+import sys
+from threading import Thread
+from concurrent.futures import ThreadPoolExecutor
+
+if len(sys.argv) == 2:
+    ip = sys.argv[1]
+    print("Sentinel IP used %s" % ip)
+else:
+    ip = "127.0.0.1"
+
+
+def safe_print(content):
+    print("{0}".format(content))
+
+
+class updateRedis():
+
+    def __init__(self, ip="127.0.0.1", port="16379"):
+
+        r = redis.Redis(host=ip, port=port, db=0)
+        self._ip, self._port = r.sentinel_get_master_addr_by_name('scality-s3')
+
+    def read(self, ressource, name):
+
+        r = redis.Redis(host=self._ip, port=self._port, db=0)
+        store = r.get('s3:'+ressource+':'+name+':storageUtilized:counter')
+        nbr = r.get('s3:'+ressource+':'+name+':numberOfObjects:counter')
+        print("Redis:%s:%s:%s" % (name,int(nbr),int(store)))
+
+    def update(self, ressource, name, size, files):
+
+        timestamp = int(time.time() - 15 * 60) * 1000
+        r = redis.Redis(host=self._ip, port=self._port, db=0)
+
+        numberOfObjects = 's3:%s:%s:numberOfObjects' % (ressource, name)
+        storageUtilized = 's3:%s:%s:storageUtilized' % (ressource, name)
+
+        r.zremrangebyscore(numberOfObjects, timestamp, timestamp)
```

Comment:
just curious, why are sorted sets used for these stats?

Reply:
Does this section in the Design doc help answer that?

https://github.com/scality/utapi/blob/development/8.0/DESIGN.md#sorted-sets

Reply:
nice ty!